slurm-docker-cluster/README.md

```diff
@@ -0,0 +1,104 @@
+# Slurm Docker Cluster
+
+This is a multi-container Slurm cluster using docker-compose.  The compose file
+creates named volumes for persistent storage of MySQL data files as well as
+Slurm state and log directories.
```

Comment:
There's no more docker-compose file in this folder anymore, should we link to the webknossos-libs repo?

Reply:
cc @philippotto

Reply:
Good point. I just pushed https://github.com/scalableminds/dockerfiles/commit/01d1f9d1f1c34f20df167f3abad058efd9eff4f0.